FifthLesson/VectorProgram/VectorProgramTests/ValidatorTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VectorProgram.Controller;
+using VectorProgram.Model;
+
+namespace VectorProgramTests
+{
+    [TestClass]
+    public class ValidatorTests
+    {
+        [TestMethod]
+        public void ValidateInput_CorrectValues()
+        {
+            bool actual;
+
+            var inputMultiplier = new[] { "1", "56565", "1.0", "1,54" };
+            var inputVector = new[] { " 1 1 1", "0,005 0,005 0,005", "0.549 0.549 0.549", "1 1 1" };
+
+            foreach (var item in inputMultiplier)
+            {
+                actual = Validator.ValidateInput(DataType.Multiplier, item);
+                Assert.AreEqual(true, actual);
```

Comment:
You can use _Assert.IsTrue_ to check whether the value is true

Reply:
Corrected.